scalability: "Average CPU usage" flakes

<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!

If the matter is security related, please disclose it privately via https://kubernetes.io/security/
-->

/kind flake
**What happened**:
The scalability test failed on the "Average CPU usage 504mCpu is greater than maximum expected 500mCPU".

```shell
{Failed  === RUN   TestScalability/CommandStats
    checker_test.go:87: Average CPU usage 504mCpu is greater than maximum expected 500mCPU
--- FAIL: TestScalability/CommandStats (0.00s)
}
```

**What you expected to happen**:
No error happened.

**How to reproduce it (as minimally and precisely as possible)**:
https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_kueue/2235/pull-kueue-test-scheduling-perf-main/1792493875051368448

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`):
- Kueue version (use `git describe --tags --dirty --always`):
- Cloud provider or hardware configuration:
- OS (e.g: `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

--- a/test/performance/scheduler/default_rangespec.yaml
+++ b/test/performance/scheduler/default_rangespec.yaml
@@ -8,8 +8,8 @@ cmd:
   # Average value 351116.4 (+/- 0.9%), setting at +20%
   maxWallMs: 425_000
 
-  # Average value 396 mCPU (+/- 8%), setting at +25%
-  mCPU: 500
+  # Average value 396 mCPU (+/- 8%), setting at +30%
+  mCPU: 515
   
   # Average value 445012 (+/- 0.3%), setting at +5%
   maxrss: 468_000

